src/harness/BenchmarkDotNet.Extensions/ValuesGenerator.cs

```diff
@@ -182,5 +184,23 @@ private static Guid GenerateRandomGuid(Random random)
             random.NextBytes(bytes);
             return new Guid(bytes);
         }
+
+        private static decimal GenerateRandomDecimal(Random random)
```

Comment:
This is the [Jon Skeet method](https://stackoverflow.com/a/609529) cleaned up to reflect the style in this codebase. I don't think that the fact that these values are not uniformly distributed should matter here (given how we use a subset of ASCII in the string tests), but if desired can [Rasmus Faber variant](https://stackoverflow.com/a/610228)

Reply:
Addressed desire for commenting WHY in the code.